src/main/java/Task.java

```diff
@@ -0,0 +1,34 @@
+public class Task {
+    private boolean isDone;
+    private String description;
+
+    public Task() {
+        this(false,"");
+    }
+
+    public Task(boolean status, String s){
```

Comment:
Perhaps a more descriptive variable name such as "description" instead of "s"? I noticed a lot of other single-letter named variables used too.